src/mozillavpn.cpp

```diff
@@ -134,7 +133,7 @@ MozillaVPN::MozillaVPN() : App(nullptr), m_private(new MozillaVPNPrivate()) {
   connect(this, &MozillaVPN::stateChanged, [this]() {
     // If we are activating the app, let's initialize the controller and the
     // periodic tasks.
-    if (state() == StateMain) {
+    if (state() == StateMain || state() == StateOnboarding) {
```

Comment:
Why this update? Because we need to be able to start the connection during onboarding now? Why didn't we need this for desktop, out of curiousity?

Reply:
With the current implementation, we need the connection manager initialized in onboarding as a means to present the "Add VPN configuration" system modal (starting with `ConnectionManager::activate()`). We do not have this requirement on desktop.

Reply:
Makes sense, thanks.